src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/ClientSettingsStorageLocation.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Options;
+
+/// <summary>
+/// Specifies that the option is stored in setting storage of the client.
```

Comment:
I admit this comment is a bit vague -- is the difference here unlike the old names this may or may not roam, as opposed to the local one that is most definitely not-roaming?

Reply:
Or perhaps phrased another way: is this more a rename or is there some deeper meaning here? To be clear I'm not thinking anything is wrong here, as much as I'm just trying to understand direction.

Reply:
> is the difference here unlike the old names this may or may not roam, as opposed to the local one that is most definitely not-roaming

Yes, it's a base class for both roaming and non-roaming options stored on the client.